data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/extension/GeoIPProcessorService.java

```diff
@@ -41,19 +37,19 @@ public class GeoIPProcessorService {
     private static final Logger LOG = LoggerFactory.getLogger(GeoIPProcessorService.class);
     public static final String DATABASE_1 = "first_database_path";
     public static final String DATABASE_2 = "second_database_path";
-    private static final String TEMP_PATH_FOLDER = "GeoIP";
-    private GeoIPProcessorConfig geoIPProcessorConfig;
-    private LicenseTypeOptions licenseType;
+    private static final String TEMP_PATH_FOLDER = "geoip";
```

Comment:
Should this instance of "GeoIP" be changed to "geoip" as well:
https://github.com/opensearch-project/data-prepper/blob/2be8166ac8ac1f80784201f2a6e33302145cb5a8/data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/databasedownload/DBSource.java#L29
 Looks to me the two are referring to the same directory.

Reply:
Yeah looks like it's hard cded here, I will remove it in next PR. This won't be used in upcoming change